pkg/dependencies/version.go

```diff
@@ -24,7 +24,7 @@ var (
 	clangRegex = regexp.MustCompile(`(?m)clang version (?P<version>\d+\.\d+(\.\d+)?)`)
 	cmakeRegex = regexp.MustCompile(`(?m)cmake version (?P<version>\d+\.\d+(\.\d+)?)`)
 	llvmRegex  = regexp.MustCompile(`(?m)LLVM version (?P<version>\d+\.\d+(\.\d+)?)`)
-	javaRegex  = regexp.MustCompile(`(?m)version \"(?P<version>\d+(\.\d+\.\d+)?)(_\d+)?\"`)
+	javaRegex  = regexp.MustCompile(`(?m)version "(?P<version>\d+(\.\d+\.\d+)*)\.*\d*?(_\d+)?"`)
```

Comment:
```suggestion
	javaRegex  = regexp.MustCompile(`(?m)version "(?P<version>\d+(\.\d+\.\d+)*)([_\.]\d+)?"`)
```
IMO this is easier to understand (as far as this is possible for regex)
So its read basically as `semver` optionally followed by `_number` or `.number`

Reply:
Cool, thanks!